tests/skipCrossGenFiles.arm.txt

```diff
@@ -0,0 +1,2 @@
+mscorlib.dll
+System.Runtime.WindowsRuntime.UI.Xaml.dll
```

Comment:
In fact, `mscorlib.ni.dll` and `System.Private.CoreLib.ni.dll` cannot be generated in build time for arm32/linux yet. But we should generate these files first to test crossgen in target device, so it seems reasonable.